crates/accelerate/src/remove_identity_equiv.rs

```diff
@@ -33,12 +34,15 @@ fn remove_identity_equiv(
 ) {
     let mut remove_list: Vec<NodeIndex> = Vec::new();
     let mut global_phase_update: f64 = 0.;
+    // Minimum threshold to compare average gate fidelity to 1. This is chosen to account
+    // for roundoff errors and to be consistent with other places.
+    let minimum_tol = 1e-12_f64;
```

Comment:
I'd make this a `const`:

```suggestion
    const MINIMUM_TOL: f64 = 1e-12;
```

Although you might have to move it outside the function definition to do this.